src/host/premake.h

```diff
@@ -62,6 +62,8 @@
 #define PLATFORM_ARCHITECTURE "ARM"
 #elif defined(_M_RISCV64) || (defined(__riscv) && __riscv_xlen == 64)
 #define PLATFORM_ARCHITECTURE "RISCV64"
+#elif (defined(__loongarch__) && __loongarch_grlen == 64) || defined(__loongarch64)
```

Comment:
`|| defined(__loongarch64)` is not needed here, it's deprecated.

Reply:
This has already been merged. If there is an issue here, I would appreciate a follow on PR. I do not have a computer with this architecture to verify builds on.

Reply:
> This has already been merged. If there is an issue here, I would appreciate a follow on PR. I do not have a computer with this architecture to verify builds on.

Oh, I see.

This will not affect functionality, but only some predefined macro code specifications. If you are okay with this, it is fine to keep it as is.